### DISCUSSION TITLE
Getting errors in Hasura and Graphite after disabling AI in settings

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#2693] [Q&A]
We turned off our AI Service and yet it still is running in the background and we are getting all sorts of errors, like the one below and our metadata is totally broken in Hasura. How do we stop the service from actually running and get rid of these errors? Just disabling it, doesn't seem to do anything.  

{"detail":{"http_info":{"content_encoding":null,"http_version":"HTTP/1.1","ip":"","method":"POST","status":200,"url":"/v1/graphql"},"operation":{"error":{"code":"validation-failed","error":"field 'graphite' not found in type: 'query_root'","path":"$.selectionSet.graphite"},"query":{"type":null},"request_id":"401e9c116fffbdbbc7c7e21231f83e3c","request_mode":"error","response_size":120,"uncompressed_response_size":120,"user_vars":{"x-hasura-role":"admin"}},"request_id":"401e9c116fffbdbbc7c7e21231f83e3c"},"level":"error","timestamp":"2024-05-08T16:23:01.206+0000","type":"http-log"}

**Comment by dbarrosop** [comment]
that message is actually indicating the opposite, notice the error says that an admin is trying to access the ai service data but it doesn't exist (might simply be the dashboard checking if the service is running to enable the AI tab)

if you want to check if the service is running easiest would be to go to the logs tab and filter by the ai service

**Reply by osseonews** [reply]
OK, not sure what the error meant, but why is anything posting to that URL if the AI service is turned off? Also, this is the error in Hasura. Maybe it's related? How do we deal with this error? Just delete that metadata?
![metadata](https://github.com/nhost/nhost/assets/26583659/b2f6e7c4-96ba-4194-bc2e-e4d1231cf543)

**Reply by dbarrosop** [reply]
yes, exactly. Will add a note to the docs to avoid confusion. Thanks for reporting.
